src/components/VAudioTrack/layouts/VRowLayout.vue

```diff
@@ -75,11 +76,13 @@
           name="play-pause"
           :size="isLarge ? 'medium' : 'large'"
           :layout="'row'"
+          :is-tabbable="false"
```

Comment:
In RowLayout on line 54 we also need to add `true` to `timeFmt`.

Reply:
Fixed in commit e3dd1eb.